plugins/mcreator-localization/lang/texts.properties

```diff
@@ -815,6 +815,7 @@ blockly.block.compare_mcitems_oredictionary=Is %1 tagged in item tags as %2
 blockly.block.console_log=Log %1 with log level %2
 blockly.block.copy_mcitem=Make a copy of %2 %1
 blockly.block.deal_damage=Deal %1 damage to %2 type:
+blockly.block.damagesource_isequalto=Is provided damage source equal to %1
```

Comment:
I'd also change this text to "Is provided damage source of type %1", again to follow the 1.19.4 logic